### PR TITLE
Update Chain Registry for ixo Impact Hub

### PR DESCRIPTION
**Pull request by ig-shaun**

### CODE REVIEW

File: ixo/chain.json [hunk 1]
```diff
@@ -3,9 +3,35 @@
     "chain_name": "ixo",
```
Comment (sunnya97) [inline ixo/chain.json:3]: Should the chain name also be `impacthub` then?

File: ixo/chain.json [hunk 2]
```diff
@@ -3,9 +3,35 @@
...
     "chain_id": "impacthub-3",
     "bech32_prefix": "ixo",
+    "daemon_name": "ixod",
+    "node_home": "$HOME/.ixod",
+    "genesis": {
+        "genesis_url": "https://github.com/ixofoundation/genesis/raw/master/impacthub-3/genesis.json"
+    },
+    "key_algos": [
+        "secp256k1",
+        "ED25519"
```
[the remark this answers is not in the document]
Comment (sunnya97) [reply ixo/chain.json:16]:
```suggestion
        "ed25519"
```

[the remark this answers is not in the document]
Comment (ig-shaun) [reply ixo/chain.json]: Yes I confirm we have on-chain ed25519 as well as secp256 for on-chain accounts

File: ixo/chain.json (hunk 1 above)
Comment (ig-shaun) [reply ixo/chain.json:3]: Yes you are correct

File: ixo/chain.json [hunk 3]
```diff
@@ -3,9 +3,39 @@
     "chain_name": "ixo",
```
Comment (sunnya97) [inline ixo/chain.json:3]:
```suggestion
    "chain_name": "impacthub",
```

File: ixo/chain.json [hunk 4]
```diff
@@ -3,9 +3,39 @@
...
+        "recommended_version": "v1.6.0",
+        "compatible_versions": [
+            "v1.6.0"
+        ],
+        "binaries": {
+            "linux/amd64": "https://github.com/ixofoundation/ixo-blockchain/releases/download/v1.6.0/ixod-1.6.0-linux-amd64",
+            "linux/arm64": "https://github.com/ixofoundation/ixo-blockchain/releases/download/v1.6.0/ixod-1.6.0-linux-arm64",
+            "darwin/amd64": "https://github.com/ixofoundation/ixo-blockchain/releases/download/v1.6.0/ixod-1.6.0-darwin-amd64",
+            "windows/arm64": "https://github.com/ixofoundation/ixo-blockchain/releases/download/v1.6.0/ixod-1.6.0-windows-amd64.exe"
+        }
```
Comment (sunnya97) [inline ixo/chain.json:38]:
```suggestion
        },
```

File: ixo/chain.json [hunk 5]
```diff
@@ -1,11 +1,41 @@
...
+        "recommended_version": "v1.6.0",
+        "compatible_versions": [
+            "v1.6.0"
+        ],
+        "binaries": {
+            "linux/amd64": "https://github.com/ixofoundation/ixo-blockchain/releases/download/v1.6.0/ixod-1.6.0-linux-amd64",
+            "linux/arm64": "https://github.com/ixofoundation/ixo-blockchain/releases/download/v1.6.0/ixod-1.6.0-linux-arm64",
+            "darwin/amd64": "https://github.com/ixofoundation/ixo-blockchain/releases/download/v1.6.0/ixod-1.6.0-darwin-amd64",
+            "windows/arm64": "https://github.com/ixofoundation/ixo-blockchain/releases/download/v1.6.0/ixod-1.6.0-windows-amd64.exe"
+        },
```
Comment (sunnya97) [inline ixo/chain.json:38]:
```suggestion
        }
    },
```

**Pull request merged**